common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js

```diff
@@ -121,6 +132,11 @@ define('video/09_events_plugin.js', [], function() {
             return player ? player.currentTime : 0;
         },
 
+        getCurrentLanguage: function() {
+            var player = this.state.lang;
```

Comment:
I was following the method used in the `getCurrentTime` function. Happy to update if you think it makes sense.